<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>getRepresentation</class>
 <widget class="QDialog" name="getRepresentation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select resource...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Representation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rpNone">
          <property name="text">
           <string>None</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rpImage">
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rpModel">
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rpRectangle">
          <property name="text">
           <string>Rectangle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rpPath">
          <property name="text">
           <string>Vector path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rpGraph">
          <property name="text">
           <string>Graph</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rpText">
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pgNone">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Select a representation for this object&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;This is only used for the Editors.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Render events will ignore this setting runtime.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgImage">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Select the image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="movement">
            <enum>QListView::Static</enum>
           </property>
           <property name="flow">
            <enum>QListView::LeftToRight</enum>
           </property>
           <property name="resizeMode">
            <enum>QListView::Adjust</enum>
           </property>
           <property name="layoutMode">
            <enum>QListView::SinglePass</enum>
           </property>
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
            <property name="icon">
             <iconset resource="../../sharedcode/resources.qrc">
              <normaloff>:/resources/RES/app.png</normaloff>:/resources/RES/app.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/resources/RES/defbg.jpg</normaloff>:/resources/RES/defbg.jpg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
            <property name="icon">
             <iconset resource="../../sharedcode/resources.qrc">
              <normaloff>:/resources/RES/clear_left.png</normaloff>:/resources/RES/clear_left.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
            <property name="icon">
             <iconset resource="../../sharedcode/resources.qrc">
              <normaloff>:/resources/RES/app.png</normaloff>:/resources/RES/app.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgModel"/>
       <widget class="QWidget" name="pgRectangle"/>
       <widget class="QWidget" name="pgVectorPath"/>
       <widget class="QWidget" name="pgText"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../sharedcode/resources.qrc"/>
  <include location="../../sharedcode/resources.qrc"/>
  <include location="../../sharedcode/resources.qrc"/>
  <include location="../../sharedcode/resources.qrc"/>
  <include location="../../sharedcode/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>getRepresentation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>getRepresentation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
